src/arkode/arkode_ls.c

```diff
@@ -3362,14 +3357,16 @@ int arkLs_AccessLMem(void* arkode_mem, const char* fname, ARKodeMem* ark_mem,
   void* ark_step_lmem;
   if (arkode_mem == NULL)
   {
-    arkProcessError(NULL, ARKLS_MEM_NULL, "ARKLS", fname, MSG_LS_ARKMEM_NULL);
```

Comment:
The input `fname` is no longer used and could be removed.

Reply:
This line youre referring to was a line that was removed.

Reply:
See this [comment](https://github.com/LLNL/sundials/pull/381#discussion_r1421126771)